modules/ui/commit_warnings.js

```diff
@@ -1,103 +1,113 @@
 import { select as d3_select } from 'd3-selection';
-
 import { t } from '../core/localizer';
 import { svgIcon } from '../svg/icon';
-import { uiTooltip } from './tooltip';
+import { uiSection } from '../ui/section';
 import { utilEntityOrMemberSelector } from '../util';
 
-
 export function uiCommitWarnings(context) {
+    var _issuesBySeverity = {};
 
     function commitWarnings(selection) {
-        var issuesBySeverity = context.validator()
+        // Wrap the selection in a div with the class modal-section
+        selection = selection.append('div').attr('class', 'modal-section');
+
+        // Load issues by severity
+        _issuesBySeverity = context.validator()
             .getIssuesBySeverity({ what: 'edited', where: 'all', includeDisabledRules: true });
 
-        for (var severity in issuesBySeverity) {
-            var issues = issuesBySeverity[severity];
+        for (let severity in _issuesBySeverity) {
+            let issues = _issuesBySeverity[severity];
```

Comment:
good idea to refactor some of the old code using `var`. Could you please also use `const/let` instead  of `var` in the remaining new code of this PR?